### ISSUE TITLE
Progress Bar Percentages Getting Rounded Oddly

### ISSUE THREAD

**Issue by rtucker88** [react-bootstrap/react-bootstrap#396]
So I have a Progressbar on a page that gets incremented by 5 percent each time it is advanced. When it gets to 55%, 56% is displayed.

This is happening because of Math.ceil being used in the getPercentage function. Is there a reason for using ceil over round?

If we consider the common use case to be a progress bar being from 0 to 100, then we can look for inconsistencies in the output. I quickly tested the cases from 0 to 100 using Math.ceil, Math.round, and Math.floor. The results are below: 

CEIL
IN, OUT
7, 8
14, 15
28, 29
55, 56
56, 57

ROUND
IN, OUT

FLOOR
IN, OUT
29, 28
57, 56
58, 57

I'm happy to open a PR if you would like to change to Math.round.

**Comment by mathieumg** [comment]
> Is there a reason for using ceil over round?

To be optimistic I imagine? :stuck_out_tongue: 

That being said you can always leverage the `label` property if you need more fine-tuned display customization.

**Comment by mtscout6** [comment]
I think rounding down should be the default, and perhaps add a `roundUp` prop for those that care to round up. I'd be open to a pull request that supported both.

**Comment by YannPl** [comment]
The fact that percentage is rounded up causes a bug in the use of stacked bar if the max value is more than 100.

In example, try this code in the documentation example:
``` JavaScript
const progressInstance = (
  <ProgressBar>
    <ProgressBar bsStyle='success' max={1000} now={755} key={1} />
    <ProgressBar bsStyle='warning' max={1000} now={245} key={2} />
  </ProgressBar>
);

React.render(progressInstance, mountNode);
```
The first bar gets 25% and the second gets 76% so it's displayed in a second line and is invisible in the bar.

Can you please fix this either by using math.round or allow percentages to be decimals ?

Thanks a lot =)

**Comment by mtscout6** [comment]
Fixed by #532
